Formula/quilt.rb

```diff
@@ -28,12 +28,6 @@ def install
       "--prefix=#{prefix}",
       "--without-getopt",
     ]
-    on_macos do
-      args << "--with-sed=#{HOMEBREW_PREFIX}/bin/gsed"
-    end
-    on_linux do
-      args << "--with-sed=#{HOMEBREW_PREFIX}/bin/sed"
-    end
```

Comment:
This one was not to dodge the shim. It's so we use the one `gnu-sed` formula instead.